plugins/services/src/js/utils/ServiceConfigDisplayUtil.js

```diff
@@ -0,0 +1,691 @@
+import classNames from 'classnames';
+import React from 'react';
+import {Table} from 'reactjs-components';
+
+import HostUtil from './HostUtil';
+import Networking from '../../../../../src/js/constants/Networking';
+import Units from '../../../../../src/js/utils/Units';
+import Util from '../../../../../src/js/utils/Util';
+
+const renderMillisecondsFromSeconds = (prop, row) => {
+  let value = row[prop];
+
+  if (value != null) {
+    value = `${value * 1000} ms`;
+  }
+
+  return ServiceConfigDisplayUtil.getDisplayValue(value);
+};
+
+const getColumnClassNameFn = (classes) => {
+  return (prop, sortBy) => {
+    return classNames(classes, {
+      'active': prop === sortBy.prop
+    });
+  };
+};
+
+const getColumnHeadingFn = (defaultHeading) => {
+  return (prop, order, sortBy) => {
+    let caretClassNames = classNames('caret', {
+      [`caret--${order}`]: order != null,
+      'caret--visible': sortBy.prop === prop
+    });
+
+    return (
+      <span>
+        {defaultHeading || prop}
+        <span className={caretClassNames} />
+      </span>
+    );
+  };
+};
+
+const ServiceConfigDisplayUtil = {
+  displayValues: [
+    {
+      values: [
+        {
+          heading: 'General',
```

Comment:
What do you say we divide these configurations into separate files by category like it is done here: https://github.com/dcos/dcos-ui/pull/1324/files#diff-db6f93bce675a7029b88eb07056b4e52

`/DisplayConfigs/General.js`
`/DisplayConfigs/Network.js`
...
etc.

Not exactly sure if `/DisplayConfigs` is best place for those though, but something along those lines